app/assets/javascripts/components/samples.jsx

```diff
@@ -61,7 +61,7 @@ class Samples extends React.Component {
   applyClass(status) {
     if(status === 'COMPLETE') {
       return 'complete';
-    } else if (status === 'UPLOADING') {
+    } else if (status === 'UPLOADING' || status === 'IN PROGRESS') {
```

Comment:
why is this uploading? 
if 'IN PROGRESS', status should just be 'IN PROGRESS' or 'RUNNING'?

Reply:
I think job_status_descriptions UPLOADING, IN PROGRESS, FAILED AND COMPLETE, which is what is being displayed on the samples page. Rebecca suggested we have the same colors for uploading and in progress @yunfangjuan

Reply:
ok got it.